rts/Lua/LuaHandleSynced.cpp

```diff
@@ -2120,16 +2120,187 @@ int CSyncedLuaHandle::GetWatchWeaponDef(lua_State* L) {
 	return 1;
 }
 
+/*** Register/deregister callins working per defID.
+ *
+ * Some of the engine callins can result in so many callins the engine doesn't forward them until registered
+ * through the following SetWatch* methods.
+ *
+ * The GetWatch* methods can be used to query the currently registered defIDs.
+ *
+ * @section watch_methods
+ */
+
+/*** Query watch status for a unitDefID.
+ *
+ * @function Spring.GetWatchUnit
+ *
+ * @param unitDefID integer
+ * @return boolean watched Watch status.
+ *
+ * @see Spring.SetWatchUnit
```

Comment:
We don't have any convention over `@see` established yet afaik, honestly I don't care either way.